Source/Core/DolphinWX/ISOProperties/ISOProperties.cpp

```diff
@@ -231,13 +231,16 @@ void CISOProperties::CreateGUIControls()
 {
   const int space5 = FromDIP(5);
 
-  wxButton* const edit_config = new wxButton(this, ID_EDITCONFIG, _("Edit Config"));
-  edit_config->SetToolTip(_("This will let you manually edit the INI config file."));
+  wxButton* const edit_config = new wxButton(this, ID_EDITCONFIG, _("Edit User Config"));
+  edit_config->SetToolTip(
+      _("This will let you edit the user config INI file(s) for this game. Changes should be "
```

Comment:
`file` instead of `file(s)`. This only ever opens one file.

Reply:
Done